src/treeChooser/treeChooser.controller.js

```diff
@@ -82,7 +85,11 @@ function TreeChooserController(
     switch ($event.keyCode) {
       case 27: //Escape
         shouldStop = true;
-        vm.close();
+        if (vm.selected) {
```

Comment:
Tree-chooser doesn't clear the model by default. Something within our code must be clearing the model on click/focus/etc. Lets remove that instead of adding code here.